### DISCUSSION TITLE
Ability to choose larger VM's for GitHub Actions

### DISCUSSION THREAD

**Discussion by tysonzero** [community/community#16053] [Actions]
Unfortunately some of our actions are rather expensive, and are now frequently running out of memory. Some actions we would like to do are more expensive still and have never been able to succeed.

I know self hosted runners are a thing, but realistically it'll cost us _significantly_ more in both developer time and in compute costs to spin up one of those and hook it up than it would to just use a competing CI service with flexibility in VM specs.

We'd happily pay extra for bigger VM's if GitHub made them available.

**Comment by gdryke** [comment]
Check this out!
https://github.blog/2022-09-01-github-actions-introducing-the-new-larger-github-hosted-runners-beta/

**Reply by mbrevda** [reply]
Amazing! Can you whitelist an org?
